GitCommands/Git/GitModule.cs

```diff
@@ -595,6 +596,30 @@ public void SaveBlobAs(string saveAs, string blob)
             stream.Write(blobData, 0, blobData.Length);
         }
 
+        public void SaveTreeAs(string saveAs, string hash)
+        {
+            string archivePath = Path.Join(saveAs.SubstringUntilLast('\\'), $"{hash}.zip");
+
+            if (!File.Exists(archivePath))
+            {
+                GitArgumentBuilder args = new("archive")
+                {
+                    $"--format zip",
+                    $"--output \"{archivePath.ToNativePath()}\"",
+                    $"{hash}",
+                    "-0"
+                };
```

Comment:
Turn either into a `GitCommand` or move to `GitCommandHelpers`.

Reply:
Have moved to GitCommandHelpers